src/com/tw/go/plugin/git/GitCmdHelper.java

```diff
@@ -40,7 +40,8 @@ public String version() {
     @Override
     public void checkConnection() {
         CommandLine gitCmd = Console.createCommand("ls-remote", gitConfig.getEffectiveUrl());
-        runAndGetOutput(gitCmd);
+        CommandLine gitCmdMasked = Console.createCommand("ls-remote", gitConfig.getEffectiveMaskedUrl());
```

Comment:
gitCmdMasked here only for output in case of errors, to real execution we still pass gitCmd. Difference only in output while error